controllers/informationCenter/InformationCenterHandler.inc.php

```diff
@@ -104,11 +104,32 @@ function listNotes($args, $request) {
 		$templateMgr = TemplateManager::getManager($request);
 		$noteDao = DAORegistry::getDAO('NoteDAO');
 		$user = $request->getUser();
+
+		// get submission note file
+		$noteFilesDownloadLink = array();
+		$submissionFileDao = DAORegistry::getDAO('SubmissionFileDAO');
+		import('lib.pkp.classes.submission.SubmissionFile');
+		import('lib.pkp.controllers.api.file.linkAction.DownloadFileLinkAction');
+		$notes = $noteDao->getByAssoc($this->_getAssocType(), $this->_getAssocId());
+		while ($note = $notes->next()) {
+			$submissionFiles = $submissionFileDao->getLatestRevisionsByAssocId(
+				ASSOC_TYPE_NOTE, $note->getId(),
+				$this->_getAssocId(),
+				SUBMISSION_FILE_NOTE
+			);
+			if (!empty($submissionFiles)) {
+				assert(count($submissionFiles) == 1);
+				$noteFilesDownloadLink[$note->getId()] = new DownloadFileLinkAction($request, array_shift($submissionFiles), $request->getUserVar('stageId'));
+			}
+			unset($note);
```

Comment:
Now that we're no longer using references so heavily, `unset`s like this aren't needed.